addons/account/models/account_payment.py

```diff
@@ -855,12 +856,21 @@ def create(self, vals_list):
 
         payments = super().create(vals_list)
 
+        # Outstanding account should be set on the payment in community edition to force the generation of journal entries on the payment
+        # This is required because no reconciliation is possible in community, which would prevent the user to reconcile the bank statement with the invoice
+        accounting_installed = self.env['account.move']._get_invoice_in_payment_state() == 'in_payment'
+
         for i, (pay, vals) in enumerate(zip(payments, vals_list)):
             if (
                 write_off_line_vals_list[i] is not None
                 or force_balance_vals_list[i] is not None
                 or linecomplete_line_vals_list[i] is not None
+                or not accounting_installed
             ):
+                if not accounting_installed and not pay.outstanding_account_id:
+                    outstanding_account = pay._get_outstanding_account(pay.payment_type)
+                    pay.outstanding_account_id = outstanding_account.id
```

Comment:
Shouldn't we also post the move?
Normally, the code providing the `write_off_line_vals_list`/... in the create vals is responsible for posting it

Otherwise there is never a possibility to reconcile the payments made from scratch